contrib/test_jenkins.sh

```diff
@@ -371,6 +371,27 @@ build_icc() {
 	module unload intel/ics
 }
 
+#
+# Build with PGI compiler
+#
+build_pgi() {
+	echo 1..1 > build_pgi.tap
+	if module_load pgi/19.7 && pgcc18 --version
+	then
+		echo "==== Build with PGI compiler ===="
+		# PGI failed to build valgrind headers, disable it for now
+		../contrib/configure-devel --prefix=$ucx_inst CC=pgcc18 --without-valgrind
```

Comment:
pass CXX?

Reply:
CXX should be configured by module

Reply:
actually CC is configured by module too, but we are using non-default PGI compiler

Reply:
so CXX is configured by module but CC isn't?

Reply:
by default:
```
CXX=pgc++
CC=pgcc
```

but we need
```
CXX=pgc++
CC=pgcc18
```

not sure we may change module - it may be provided by PGI team
as I remember PGI is going to make pgcc18 compiler by default for C files, we may use custom compiler setting for now and remove it after module update

Reply:
okay, so let's at least document this

Reply:
ok, added TODO comment